root/src/main/java/com/example/servingwebcontent/Movie.java

```diff
@@ -1,17 +1,12 @@
 package com.example.servingwebcontent;
 
 public class Movie {
-	private String id;
```

Comment:
Why get rid of the id attribute of the Movie object? We're using autogenerated IDs but shouldn't it be ok to keep it in the business object?